pkg/controller/workload/common/fake_helpers.go

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package common
+
+import (
+	"fmt"
+	"log"
```

Comment:
use kmesh log pkg instead

Reply:
I am using `logger := logger.NewLoggerScope("workload")` like this:
```
logger := logger.NewLoggerScope("workload")
	for _, opt := range opts {
		err := opt(workload)
		if err != nil {
			logger.Errorf("Error applying option: %v", err)
		}
	}
```